jbpm-flow/src/main/java/org/jbpm/workflow/instance/node/EventSubProcessNodeInstance.java

```diff
@@ -92,7 +93,8 @@ public void nodeInstanceCompleted(org.jbpm.workflow.instance.NodeInstance nodeIn
                     }
                     getProcessInstance().getMetaData().remove("SUB_PROCESS_INTERRUPTION");
                     if (getNodeInstanceContainer() instanceof ProcessInstance) {
-                        ((ProcessInstance) getProcessInstance()).setState(ProcessInstance.STATE_ABORTED, faultName);
+                        Object faultData = getProcessInstance().getVariable("subexception");
```

Comment:
I replace it with:

    Object faultData = getProcessInstance().getVariable("event");
    
and seems to be enough. `event` is the general variable you get capturing an error

Reply:
right